modules/connectivity/locals.tf

```diff
@@ -1527,7 +1527,41 @@ locals {
       "${local.virtual_network_peering_resource_id_prefix[location]}/${peering_name}"
     }
   }
-  azurerm_virtual_network_peering = flatten(
+  virtual_network_hub_peerings = {
```

Comment:
Are we able to update this to reflect how names and resource IDs are created for the standard peering please? This allows us to offer the opportunity for user overrides through the advanced config block (something which has been explicitly asked for in a previous issue for this resource type.

Please also add new `locals` to the `debug` output